.perlcriticrc

```diff
@@ -34,6 +34,7 @@ allow=refs
 
 [Variables::ProhibitAugmentedAssignmentInDeclaration]
 [Variables::ProhibitConditionalDeclarations]
+[Variables::ProhibitPunctuationVars]
```

Comment:
This one will take me the longest to get used to since I know what all the symbols mean and don't know any of the English names without looking them up, but I can't argue that it's not more readable in the long run.

Reply:
I would hope the English names at least seem connected with the things the symbols do :)